.github/scripts/check_labels.py

```diff
@@ -42,13 +69,9 @@ def main() -> None:
     pr = GitHubPR(org, project, args.pr_num)
 
     try:
-        if not check_labels(pr.get_labels()):
-            msg = ("This PR needs a label. If your changes are user facing and intended to be a "
-                   "part of release notes, please use a label starting with `release notes:`. If "
-                   "not, please add the `topic: not user facing` label. For more information, see "
-                   "https://github.com/pytorch/pytorch/wiki/PyTorch-AutoLabel-Bot#why-categorize-for-release-notes-and-how-does-it-work.")
-            print(msg)
-            gh_post_pr_comment(pr.org, pr.project, pr.pr_num, msg)
+        if not check_labels(pr):
```

Comment:
nit: function renaming suggestion (up to you):

```suggestion
        if not has_required_labels(pr):
```